tools/wptrunner/wptrunner/executors/pytestrunner/runner.py

```diff
@@ -117,7 +117,7 @@ def record_skip(self, report):
     def record(self, test, status, message=None, stack=None):
         if stack is not None:
             stack = str(stack)
-        new_result = (test, status, message, stack)
+        new_result = (test.split("/")[-1], status, message, stack)
```

Comment:
This should perhaps be `os.path.sep`. But why do we put the test name in the subtest name at all? That just makes things fragile under renames. Can we split on `::` or whatever the separator between the path and the function name is?

Reply:
Agree on the `os.path.sep`.

As for the names of this, we currently use the file name is the metadata, https://dxr.mozilla.org/mozilla-central/source/testing/web-platform/meta/webdriver/tests/sessions/get_timeouts.py.ini#2

Changing this works might mean we have to redo all the metadata. Is that something we dont mind about (Will this break other vendors WPT?)